*** catch/internal/catch_runner_impl.hpp	2011-02-03 05:34:49.276324134 +0100
--- catch/internal/catch_runner_impl.hpp	2011-02-03 05:36:12.443924866 +0100
***************
*** 18,23 ****
--- 18,24 ----
  #include "catch_test_registry.hpp"
  #include "catch_test_case_info.hpp"
  #include "catch_capture.hpp"
+ #include "Exception.h"
  
  namespace Catch
  {
***************
*** 438,443 ****
--- 439,449 ----
              {
                  acceptMessage( ex.what() );
                  acceptResult( ResultWas::ThrewException );
+             }
+             catch( Exception& ex )
+             {
+                 acceptMessage( ex.what() );
+                 acceptResult( ResultWas::ThrewException );
              }
              catch(...)
              {
